content/docs/15-compliance.md

```diff
@@ -30,5 +34,18 @@ Spectro Cloud is certified against SOC2 Type II, compliance with the AICPA’s (
 * The annual certification of SOC2 is Independent 3rd Party Auditor. 
 * Spectro Cloud SOC 2 Type II report is available upon request for any customers or prospects with signed MNDA.
 
+**FIPS 140-2**
+
+Spectro Cloud is certified against FIPS 140-2 with Certificate number 4349 in compliance with the Cryptographic Module Validation Program (CMVP).  
```

Comment:
```suggestion
Spectro Cloud is certified against FIPS 140-2 with [Certificate number 4349](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4349) in compliance with the Cryptographic Module Validation Program (CMVP).  
```

Adding the official source as a URL to help the reader validate the information.